docs/breaking-changes.md

```diff
@@ -103,7 +103,22 @@ shell.trashItem(path).then(/* ... */)
 
 ## Planned Breaking API Changes (11.0)
 
-There are no breaking changes planned for 11.0.
+### Removed: `BrowserView.{fromId, fromWebContents, getAllViews}` and `id` property of `BrowserView`
+The experimental APIs `BrowserView.{fromId, fromWebContents, getAllViews}`
+have now been removed. Additionally, the `id` property of `BrowserView`
+has also been removed.
```

Comment:
Perhaps also mention that the `destroy` method was removed. It was never documented but I've seen a few issues of folks running into its removal.

Reply:
Sounds good, thanks! Will mention `destroy` as well